### PR TITLE
[FIX] release_docs workflow

### PR DESCRIPTION
**Pull request by man-shu**
- Closes none but tries to fix the release docs workflow

Changes proposed in this pull request:
We get a permission denied error when we try to clone the `nilearn.github.io` repo in `_build/nilearn.github.io`. 
- ~I think it's because `_build` directory does not yet exists, so create a `_build` dir first~
- it seems `git clone` can handle making a new directory if the specified cloning path does not exist. Now I think this happens because we are trying to do things in `.` and we might not have permissions to create directory there. So now we do everything in `~/`

### CODE REVIEW

**Comment by man-shu** [comment]
> * it seems `git clone` can handle making a new directory if the specified cloning path does not exist. Now I think this happens because we are trying to do things in `.` and we might not have permissions to create directory there. So now we do everything in `~/`

Update on my hypothesis for why the release docs workflow fails

File: doc/Makefile [hunk 1]
```diff
@@ -190,17 +190,18 @@ check:
 install: sym_links_datasets glm_reports
```
Comment (Remi-Gau) [inline doc/Makefile:190]:
```suggestion
install: clean sym_links_datasets glm_reports
```

File: doc/Makefile [hunk 2]
```diff
@@ -190,17 +190,18 @@ check:
...
 	@echo $(VERSIONTAG)
 	rm -rf _build/doctrees _build/nilearn.github.io
```
Comment (Remi-Gau) [inline doc/Makefile:192]:
```suggestion
```

File: doc/Makefile [hunk 3]
```diff
@@ -187,20 +187,20 @@ check:
 	rm -rf $(BUILDDIR)/html/_images
 	BUILD_DEV_HTML=1 $(SPHINXBUILD) -W --keep-going -T -n -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html
 
-install: sym_links_datasets glm_reports
+install: clean sym_links_datasets glm_reports
 	@echo $(VERSIONTAG)
-	rm -rf _build/doctrees _build/nilearn.github.io
+	mkdir ~/_build
```
Comment (Remi-Gau) [inline doc/Makefile:192]:
```suggestion
```

**Comment by Remi-Gau** [comment]
ok it is failing on my fork for other reasons.

**Comment by Remi-Gau** [comment]
but there is a workflow dispatch on this workflow so we can trigger it manually on the upstream: https://github.com/nilearn/nilearn/actions/runs/12137153863

**Comment by man-shu** [comment]
🤞🏽

**Comment by man-shu** [comment]
get_data failed so I re ran it

**Comment by Remi-Gau** [comment]
ok this is still failing

I think we should try to see if we can modify our regular doc publishing workflow to do the trick

**Pull request merged**